experimenter/experimenter/jetstream/models.py

```diff
@@ -329,71 +334,71 @@ def append_conversion_count(self, primary_metrics_set):
 """
 
 
+# XXX: does this need @lru_cache ?
 def create_results_object_model(data: JetstreamData):
-    branches = {}
-    metrics = {}
-
-    for jetstream_data_point in data:
-        branches[jetstream_data_point.branch] = {}
+    branches = [data_point.branch for data_point in data]
```

Comment:
Should this be a set? I don't think there's any reason to iterate over a list of the full length of `data` twice (which we'd be doing here and then below if `branches` is a list of the same length as `data`).

Reply:
You're right, this mistakenly changed set-like behaviour to a list.